app/middleware/parameter-validation.ts

```diff
@@ -22,6 +22,22 @@ const RANGESET_ROUTE_REGEX = /^\/.*\/ogc-api-coverages\/.*\/collections\/.*\/cov
  */
 const validLinkTypeValues = ['http', 'https', 's3'];
 
+/**
+ * Returns the bucket setup instruction 
+ *
+ * @param req - The client request
+ * @param destinationUrl - The destinationUrl
+ * @returns the bucket setup instruction
+ */
+function bucketInstruction(req: HarmonyRequest, destinationUrl: string): string {
+  const bucketPolicyUrl = `${getRequestRoot(req)}/staging-bucket-policy?bucketPath=${destinationUrl}`;
+  const instructions = [`The s3 bucket must be created in the ${awsDefaultRegion} region with 'ACL disabled' which is the default Object Ownership setting in AWS S3.`,
+    'The s3 bucket also must have the proper bucket policy in place to allow Harmony to access the bucket.',
+    'You can retrieve the bucket policy to set on the s3 bucket by calling the Harmony staging-bucket-policy endpoint with the s3 bucket path where the OGC result will be staged in. e.g.',
```

Comment:
I am planning to change this last sentence to: `You can retrieve the bucket policy to set on your s3 bucket by calling : ${bucketPolicyUrl}`